guides/micronaut-data-access-mybatis/java/src/main/java/example/micronaut/ListingArguments.java

```diff
@@ -28,7 +29,15 @@ public class ListingArguments {
     @Nullable
     private String order;
 
-    public ListingArguments() {
+    private ListingArguments() {
+    }
+
+    @Creator
```

Comment:
can we remove this `@Creator`

Reply:
Gone in https://github.com/micronaut-projects/micronaut-guides/pull/1139/commits/d748c5715e9bde0468722f8ceb5e1450aafe0862